amd64.env

```diff
@@ -1,8 +1,17 @@
-rosdistro=humble
+# ROS distribution (humble or jazzy)
+rosdistro=${ROS_DISTRO:-humble}
```

Comment:
ROS_DISTRO is usually set after we install ROS, but this file is actually used to install ROS with ansible script.
We could consider having amd64.jazzy.env instead for this file.